components/Menu.tsx

```diff
@@ -47,9 +47,10 @@ function Hamburger() {
 }
 
 export default function Menu({ items, onOpen, onClose }: Props) {
+  const router = useRouter()
   const [open, setOpen] = useState(false)
   const [active, setActive] = useState<string | null>(null)
-  const router = useRouter()
+  const isEnglish = router.asPath === "/en/"
```

Comment:
Does this only works for the "/en/" page? so a "/en/contact" page would not match?

Reply:
Yes. Fixed ;)